class.jetpack-cli.php

```diff
@@ -291,24 +310,94 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::line( __( "Resetting default modules...\n", "jetpack" ) );
 				usleep( 500000 ); // Take a breath
 				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					Jetpack::update_active_modules( $default_modules );
+				}
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 
 				// Jumpstart option is special
-				Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				if ( ! $is_dry_run ) {
+					Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
+				}
 				WP_CLI::success( __( 'jumpstart option reset', 'jetpack' ) );
 				break;
 			case 'modules':
-				$default_modules = Jetpack::get_default_modules();
-				Jetpack::update_active_modules( $default_modules );
+				if ( ! $is_dry_run ) {
+					$default_modules = Jetpack::get_default_modules();
+					Jetpack::update_active_modules( $default_modules );
+				}
+
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
+				break;
+			case 'sync-checksum':
+				$option = 'jetpack_callables_sync_checksum';
+
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites        = get_sites( array( 'number' => 1000 ) );
```

Comment:
Was `1000` added for performance reasons? What would happen on larger networks? Maybe we should add a comment explaining this number here?

Reply:
Its good to have limits in place. I am not expecting many sites to have more then 1000 sites that also have jetpack installed.

Reply:
Makes sense. Maybe add a comment here for our future selves?